client/cypress/integration/ProductPage.spec.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+
+describe('Erigolem Product Page tests', () => {
+  beforeEach(() => {
+    cy.visit('https://erigon.golem.network/');
```

Comment:
I'd prefer to avoid hard-coding production URL into tests.

Reply:
This test was removed because there is no point in checking the rendered text